submit_cgap/submission.py

```diff
@@ -293,24 +358,68 @@ def post_files_data():
 
 DEFAULT_INGESTION_TYPE = 'metadata_bundle'
 
+DEFAULT_APP = APP_CGAP
+
+
+def _resolve_app_args(institution, project, lab, award, app):
+
+    app_args = {}
+    if app == APP_CGAP:
+        required_args = {'institution': institution, 'project': project}
+        unwanted_args = {'lab': lab, 'award': award}
+    elif app == APP_FOURFRONT:
+        required_args = {'lab': lab, 'award': award}
```

Comment:
I contest these are required in all cases, unlike for CGAP, and if the loadxl ingestion is ported back to CGAP the same will be true there